raft/omdao_raft.py

```diff
@@ -844,13 +868,14 @@ def compute(self, inputs, outputs, discrete_inputs, discrete_outputs):
         outputs["yaw_period"] = outputs["rigid_body_periods"][5]
 
         # Compute some aggregate outputs manually
```

Comment:
This case_mask is required for new WEIS cases that don't run any cases in RAFT.  `max_nac_accel` was just a typo that lingered for too long.